pkg/apis/operator/v1alpha1/tektonresult_validation_test.go

```diff
@@ -36,3 +36,79 @@ func TestTektonResult_Validate(t *testing.T) {
 	err := tc.Validate(context.TODO())
 	assert.Equal(t, "invalid value: wrong-name: metadata.name, Only one instance of TektonResult is allowed by name, result", err.Error())
 }
+
+func TestTektonResultWatcherPerformancePropertiesValidate(t *testing.T) {
```

Comment:
Not a blocker, may be we can raise a follow up PR to have table-driven tests for more clarity as a further improvement